# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../luasocket/config-no-compat-5.1.patch
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./makefile.orig	2006-04-27 05:23:21.000000000 +0200
+++ ./makefile	2006-10-08 18:18:27.000000000 +0200
@@ -25,7 +25,6 @@
 	smtp.lua
 
 TO_TOP_SHARE:= \
-	$(COMPAT)/compat-5.1.lua \
 	ltn12.lua \
 	socket.lua \
 	mime.lua
